TempMail/utils/Email.swift

```diff
@@ -10,31 +10,27 @@
 import SwiftUI
 
 class Email : ObservableObject{
-    @Published var email_addr: String
-    @Published var inbox: [InboxModel]
-    var requestUrl: URLComponents
+    @Published var email_addr: String?
```

Comment:
Not the right idea conceptually since email_addr is a necessary property of the Email class. [InboxModel] is optional since an email doesn't need to have messages in the inbox. Change this to non-optional and initialize it in init()